src/pages/task/Metadata.tsx

```diff
@@ -200,6 +202,19 @@ export const Metadata: React.FC<Props> = ({ loading, data, error, taskId }) => {
             </StyledRouterLink>
           </P2>
         )}
+        {abortInfo && (
+          <P2>
+            {`Aborted by ${abortInfo.user} because of failing task: `}
```

Comment:
I don't think this label is accurate. A task can be aborted by a user for any reason, or can be automatically aborted by the system for a couple of reasons. It'll only sometimes be because a different task in the same commit queue merge failed

Reply:
Thanks for catching that. I added these two additional abort messages per the legacy UI. 
![Screen Shot 2020-11-19 at 1 30 13 PM](https://user-images.githubusercontent.com/15262143/99708881-6d172500-2a6c-11eb-879a-039b2cd5b3d6.png)
![Screen Shot 2020-11-19 at 1 30 25 PM](https://user-images.githubusercontent.com/15262143/99708890-70aaac00-2a6c-11eb-8aa7-0c2442286564.png)